packages/teuchos/parameterlist/src/Teuchos_ParameterList.hpp

```diff
@@ -909,6 +1014,24 @@ ParameterList& ParameterList::setEntry(std::string const& name_in, ParameterEntr
 }
 
 
+template<typename T>
+void ParameterList::recursivelySetValidator(
+    RCP<const ParameterEntryValidator> const& validator, int const depth){
```

Comment:
Again, to match existing formatting, can we put this `{` at the start of the next line?

Reply:
Fixed in next push